.github/workflows/build.yml

```diff
@@ -23,29 +21,20 @@ jobs:
         run: |
```

Comment:
I don't think this workflow needs jq (anymore) can it be removed entirely?

Reply:
removed!